### DISCUSSION TITLE
Switch/filter branches/tags is case-sensitive

### DISCUSSION THREAD

**Discussion by Mikaela** [community/community#57838] [Repositories]
### Select Topic Area

Bug

### Body

I often visit https://github.com/simplex-chat/simplex-chat/ on Safari iOS and I generally want to switch to the `master` branch instead of viewing the `stable` branch.

As iOS capitalises the first letter, I start searching for `Master` instead which brings up no results.

I think the branch search should be case-insensitive as I imagine it's rare for repositories to have camelCases or similar of the same branch name multiple times.

Related:

- [Linking Development Branch Search is Case-Sensitive #49532](https://github.com/orgs/community/discussions/49532) although it's opened against pull request rather than code search and navigation.

**Comment by ncimino** [comment]
there is no reason that a branch filter should be case sensitive - very annoying to have to know the case of a branch that someone else created or was perhaps created by an integration like ClickUp

a one-line fix that would make this feature better

**Reply by thisisnacho** [reply]
This is the same when running workflows manually against a particular branch through the UI. Very annoying.

**Comment by br0nstein** [comment]
It's honestly mind-boggling that this has not been fixed to this day.
